_posts/2020-10-26-dynamically-changeable-logging.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title:  "Dynamically changeable logging levels"
+date: 2020-10-26
+author: stanislav_knot
+---
+
+Logging is an indisputably important feature for applications.
+Logging monitors what the application is doing, but equally importantly,  it logs what the application is not doing. 
+Looking into application logs can help us prevent an application remaining or even getting into a faulty state.
+
+<!--more-->
+
+
+## Setting logging levels dynamically
+
+Strimzi components are split into the two groups regarding which implementation of `log4j` they use.
+
+`log4j`
+Kafka brokers
+Kafka Connect
+Kafka MirrorMaker2
+`log4j2`
```

Comment:
Should we mention why is this important? Log4j2 with build in support and Log4j without it?